_search-plugins/knn/knn-score-script.md

```diff
@@ -328,7 +331,8 @@ A space corresponds to the function used to measure the distance between two poi
 
 Cosine similarity returns a number between -1 and 1, and because OpenSearch relevance scores can't be below 0, the k-NN plugin adds 1 to get the final score.
 
-With cosine similarity, it is not valid to pass a zero vector (`[0, 0, ...`]) as input. This is because the magnitude of
-such a vector is 0, which raises a `divide by 0` exception in the corresponding formula. Requests 
-containing the zero vector will be rejected and a corresponding exception will be thrown.
-{: .note }
\ No newline at end of file
+With cosine similarity, it is not valid to pass a zero vector (`[0, 0, ... ]`) as input. This is because the magnitude of such a vector is 0, which raises a `divide by 0` exception in the corresponding formula. Requests containing the zero vector will be rejected and a corresponding exception will be thrown.
```

Comment:
```suggestion
With cosine similarity, it is not valid to pass a zero vector (`[0, 0, ... ]`) as input. This is because the magnitude of such a vector is 0, which raises a `divide by 0` exception in the corresponding formula. Requests containing the zero vector will be rejected, and a corresponding exception will be thrown.
```